### DISCUSSION TITLE
Datadog scaler - scaling to predefined desired replica count

### DISCUSSION THREAD

**Discussion by taliv-upstream** [kedacore/keda#3069] [Q&A / Need Help]
Hi,
I am testing the DataDog scaler, we have a unique case, we want to scale our pods, but the scaling is to pre-known replicas. Like 2,4 and 8. And that will always be the transition, for example, scale from 2 to 4 and then 4 to8, 8 to 4, etc based on the metric value coming from datadog. I know that the underlying HPA object doesn't work like this because it computes the desired replica count, but I was still curious if it is something achievable in the DataDog scaler.

Thank you!

**Comment by zroubalik** [comment]
Unfortunately this is not something we can do at the moment, we would have to fake the metrics to HPA.
There has been a similar request for Kafka Scaler though https://github.com/kedacore/keda/issues/2581
